packages/frames.js/src/core/createFrames.ts

```diff
@@ -61,10 +75,12 @@ export function createFrames<
      */
     return async function handleFramesRequest(request: Request) {
       const context: FramesContext<TState> = {
+        baseUrl: url,
         basePath,
         initialState: initialState as TState,
         request,
         url: new URL(request.url),
+        resolvedBaseUrl: resolveBaseUrl(request, url, basePath),
       };
```

Comment:
is it necessary to expose all of these (`baseUrl`, `basePath`, `url`, and `resolvedBaseUrl`) to the user? perhaps we should just give them the `resolvedBaseUrl` and call it `baseUrl` and remove the `url` shortcut to be more explicit?

Reply:
`url` was there because @davidfurlong found it better to access it like this. So I would not remove that. I will move `resolvedBaseUrl` to `baseUrl`.

Reply:
i don't think basePath was very useful to the user to begin with so it's okay to remove in a breaking change. what do you think?

Reply:
Lets do that in separate change